azure/spoke/outputs.tf

```diff
@@ -40,11 +40,11 @@ output "routes" {
 
 output "network_watcher" {
   description = "The output of the Network Watcher."
-  value       = azurerm_network_watcher.main
+  value       = one(azurerm_network_watcher.main)
 }
 
 output "flow_log" {
-  value = module.network_security_group[0].flow_log
+  value = try(one(module.network_security_group[0].flow_log), null)
```

Comment:
There's already a `one()` function in the NSG module on the output of `flow_log`. `one()` function expects a list/set input but the NSG module outputs an object

https://github.com/ans-coe/terraform-modules/blob/main/azure/network-security-group/outputs.tf#L43

Reply:
Because of this, and because of the `try()` when running the advanced example, the output is `null`

```
> module.spoke.flow_log
null
```